joomla/joomla-cms
# [5.2] Cache: Fix for counting the Number of Files correctly

Issue: The ``Maintenance: Clear Cache`` page does not display the correct number + size of cached files.

This PR corrects the ``Number of Files`` on the ``Maintenance: Clear Cache`` page.

### Summary of Changes
When using Joomla's ``System - Page Cache`` plugin, an ``/administrator/cache/page`` folder with ``index.html`` file is created. When you visit one page on the front-end, you will cache **one page**. 
However, the ``Maintenance: Clear Cache`` page **shows 2 files** under ``Number of Files`` because it also counts the ``index.html`` file (which is not a cached file). It's confusing to see that number.


### Testing Instructions
1. Switch on Joomla's ``System - Page Cache`` plugin

![system-page-cache](https://github.com/user-attachments/assets/d86add92-bfdc-4e42-9e13-b04c27e29793)

2. Goto the front-end and open one page. The ``Maintenance: Clear Cache`` counts 2 files.
3. Open the website files, e.g. using an FTP client like FileZilla.
You'll see just one cached file and the ``index.html``.

![files](https://github.com/user-attachments/assets/9389446e-3b2e-41b9-a7fd-693bfefa92c8)

### Actual result BEFORE applying this Pull Request

You'll see too many cached files under ``Number of Files`` (and Size) because it includes the ``index.html``:  

![maintenance-clear-cache](https://github.com/user-attachments/assets/e67357df-f7d7-4495-a57e-933965b2add0)

### Expected result AFTER applying this Pull Request

You'll see the real number of cached files + size because the ``index.html`` is not counted as cache anymore:  

![after-pr](https://github.com/user-attachments/assets/b8f41b19-15d8-4865-aff1-caa1962f7ec1)

### Link to documentations
Please select:
